apps/dotcom/sync-worker/src/TLPostgresReplicator.ts

```diff
@@ -25,6 +26,8 @@ CREATE TABLE user_file_subscriptions (
 );
 `
 
+const ONE_MINUTE = 60 * 1000
```

Comment:
We could also push an event for each postgres update, but this feels sufficient for now.